lib/MwbExporter/Formatter/Doctrine2/Annotation/Model/Column.php

```diff
@@ -131,6 +135,8 @@ public function writeGetterAndSetter(WriterInterface $writer)
                 $this->writePrimaryKeyFetcher($writer, $nativeType);
```

Comment:
is this still required? generates additional field over and above "getId" (called fetchId)

Reply:
I've left it in as fetchId was used by Dom in integration of the event store so we'll remove when he can adjust that and switch over to getId vs fetchId.

Reply:
Ok - in that case can we adjust to add in the "fetchId" string?